src/LogicalCameraSensor.cc

```diff
@@ -104,18 +110,28 @@ bool LogicalCameraSensor::Load(sdf::ElementPtr _sdf)
     return false;
 
   if (this->Topic().empty())
-    this->SetTopic("/logical_camera");
+    this->SetTopic("/camera/logical");
```

Comment:
@BA-Utkarsh 
I'd like to know why you change a topic name of logical camera.

Reply:
We are re-visiting the changes and update soon.

Reply:
LGTM